### PR TITLE
AMReX/pyAMReX/PICSAR: weekly update

### PR DESCRIPTION
**Pull request by EZoni**
- Weekly update to latest AMReX:
```console
./Tools/Release/updateAMReX.py
```
- Weekly update to latest pyAMReX:
```console
./Tools/Release/updatepyAMReX.py
```
- Weekly update to latest PICSAR (no changes since 24.09):
```console
./Tools/Release/updatePICSAR.py
```

### CODE REVIEW

**Comment by EZoni** [comment]
@WeiqunZhang @atmyers 

Any change on the AMReX end that could explain the CUDA and HIP errors seen here?

**Comment by EZoni** [comment]
@WeiqunZhang @atmyers 

I think it is related to https://github.com/AMReX-Codes/amrex/pull/4259, we probably need to update our dependencies to include packages that were not needed before. I will give it a try following what is done in https://github.com/AMReX-Codes/amrex/pull/4259.

File: .github/workflows/dependencies/nvcc11-3.sh [hunk 1]
```diff
@@ -41,7 +41,8 @@ sudo apt-get install -y          \
     cuda-nvml-dev-11-3           \
     cuda-nvtx-11-3               \
     libcufft-dev-11-3            \
-    libcurand-dev-11-3
+    libcurand-dev-11-3           \
+    libcusparse-dev-11-3
```
Comment (RemiLehe) [inline .github/workflows/dependencies/nvcc11-3.sh:45]: @WeiqunZhang @atmyers Does this mean that installation instructions for e.g. Perlmutter should be updated to make sure that `cusparse` is installed?
https://warpx.readthedocs.io/en/latest/install/hpc/perlmutter.html

**Comment by WeiqunZhang** [comment]
Yes, I pinged you here. https://github.com/ECP-WarpX/WarpX/pull/5493#issuecomment-2542046917

File: .github/workflows/dependencies/nvcc11-3.sh (hunk 1 above)
Comment (WeiqunZhang) [reply .github/workflows/dependencies/nvcc11-3.sh:45]: It's part of the cuda package already installed on perlmutter. I don't we need anything special.

**Comment by EZoni** [comment]
Ah cool, thanks @WeiqunZhang. I think I had missed that ping.

**Review by WeiqunZhang — APPROVED** [review]

**Pull request merged**